### DISCUSSION TITLE
customizing the application upon request

### DISCUSSION THREAD

**Discussion by EdwinAndabwa** [frappe/books#825] [General]
Can someone assist in customizing the application upon request? The issues to be addressed are as follows:

(a) Transform it into an online app with multi-user functionality.
(b) Incorporate features for cash flow and the statement of changes in equity.
(c) Integrate a comparison tool for the previous and current year's financial statements (profit and loss account, balance sheet, cash flow, statement of changes in equity).
(d) Revise the format and layout of invoices, receipts, and reports. Enable online sending of invoices, receipts, and reminders.
(e) Add an asset register and depreciation using the straight-line method. The amount of depreciation should be reflected in the trial balance. This requires accounting knowledge. If you are interested, please reach out to me via eandabwa4@gmail.com.

**Comment by Isaac-GC** [comment]
Hi @EdwinAndabwa!

Currently, frappe books is only a desktop application, and while there are features requests for the same above asks, it will be a little while before all of them can be implemented. 

From what I understand you are asking, I would recommend looking into ERPNext https://github.com/frappe/erpnext
